src/Vts.Test/IO/CustomBinaryStreamWriterOfTTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using Vts.IO;
+
+namespace Vts.Test.IO
+{
+    /// <summary>
+    /// Stream finder tests
+    /// </summary>
+    [TestFixture]
+    public class CustomBinaryStreamWriterOfTTests
+    {
+        /// <summary>
+        /// list of temporary files created by these unit tests
+        /// </summary>
+        readonly List<string> listOfTestGeneratedFolders = new List<string>()
```

Comment:
Suggested name of a private field is _listOfTestGeneratedFolders and should be explicitly marked as private.

The type specification is not needed: new()